app/Models/SnipeSCIMConfig.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Helper;
+use ArieTimmerman\Laravel\SCIMServer\SCIM\Schema;
+use ArieTimmerman\Laravel\SCIMServer\Attribute\AttributeMapping;
+
+
+class SnipeSCIMConfig extends \ArieTimmerman\Laravel\SCIMServer\SCIMConfig
+{
+    public function __constructor(...$params)
+    {
+        \Log::info("CONSTRUCTOR CALLED!!@!!!! FOr my config thing");
+        self::super(...$params);
+    }
+
+    public function getUserConfig()
+    {
+        // Much of this is copied verbatim from the library, then adjusted for our needs
```

Comment:
Is it possible to use this as a trait instead? (Might require a PR against them, but would be arguably better than copypasta)

Reply:
I mean, *maybe* we could call `parent::getUserConfig()` (or `super::`) and get the result, and just add our few couple of twiddlings, relative to the library, and then return that?

Yeah, actually, that’s a pretty good idea, and keeps the change smaller, and highlights the Snipe-IT-specific bits that we’re doing. I’ll do that.

Reply:
Yup! Done :)